### PR TITLE
PIL-1245 changed walletConnect tab nav color to purple

### PR DESCRIPTION
**Pull request by vignesha22**
image
image

### CODE REVIEW

File: src/components/layout/TabBar.js [hunk 1]
```diff
@@ -98,5 +98,5 @@ const Underline = styled.View`
   border-top-width: ${Platform.OS === 'android' ? '3px' : '6px'};
   border-top-left-radius: 3px;
   border-top-right-radius: 3px;
-  border-color: ${({ theme }) => theme.colors.primaryAccent130};
+  border-color: #A020F0;
```
Comment (IAmKio) [inline src/components/layout/TabBar.js:101]: Could you check that these colour is not stored in the theme?

**Review by IAmKio — CHANGES_REQUESTED** [review]
@vignesha22 please check that this hex is stored in the theme files first.

File: src/components/layout/TabBar.js (hunk 1 above)
Comment (vignesha22) [reply src/components/layout/TabBar.js:101]: yeah i checked and one is there on the dark theme but its not there on light theme. but we needed the same color irrespective on the dark mode or light mode right?

Comment (IAmKio) [reply src/components/layout/TabBar.js:101]: It looks this way yes. If it exists for dark but not light, please add it for the light theme and use the theme reference instead.

Comment (vignesha22) [reply src/components/layout/TabBar.js:101]: yeah okay

Comment (IAmKio) [reply src/components/layout/TabBar.js]: Thanks 🙏

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**